index.html

```diff
@@ -340,3 +340,4 @@ <h2>Skills</h2>
     </footer>
```

Comment:
- [ ] The Menu is good however when the user clicks the about option on the mobile menu the contact is displayed, please consider fixing this so your project can be approved

Reply:
- [ ] As suggested by the previous reviewer kindly fix the about the option when clicked should display the about page. To fix this kindly remove this [line number 46](https://github.com/dunnyk/personal-portfolio/blob/mobile-menu/style.css#L46) of code from CSS.